server/test/snippet.test.js

```diff
@@ -71,7 +75,7 @@ describe('Snippets routes', function() {
 
     it('should return array of snippets', function (done) {
       request(app)
-        .get('/snippets')
+        .get('/snippets?year='+year+'&month='+month)
```

Comment:
I won't insist but we have this feature available in node 4 :+1: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals